tests/python/gaia-ui-tests/testvars/testvars.json

```diff
@@ -0,0 +1,15 @@
+{  
```

Comment:
As this file will contain passwords, we don't commit it. We have another hidden place where we store 1 testvars.json per device.